CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
+## 0.11.5
+- Add common properties argument to WSGIApplication initialization.
```

Comment:
We haven't shipped `11.4` yet. So it should be reported under previous title. I'll merge and fix this